src/VisualStudio/Core/Test/CodeModel/CSharp/EventCollectorTests.vb

```diff
@@ -786,7 +786,7 @@ class C
 </Code>
 
             Test(code, changedCode,
-                 ArgChange("System.CLSCompliant"))
+                 ArgChange("System.CLSCompliant", "foo"))
```

Comment:
Both these testcases already have a bunch of them. This will requrie a much bigger cleaning